devicetypes/smartthings/zwave-metering-switch.src/zwave-metering-switch.groovy

```diff
@@ -145,13 +145,18 @@ def parse(String description) {
 def handleMeterReport(cmd){
 	if (cmd.meterType == 1) {
 		if (cmd.scale == 0) {
+			//automatically updates Watts on GoControl Plug-in Switch
+			if(zwaveInfo.mfr == "014F" && zwaveInfo.prod == "5053" && zwaveInfo.model == "3531") {
+				sendHubCommand(encap(meterGet(scale: 2)))
```

Comment:
Are you doing this because the switch isn't reporting power, because this won't work with local execution. Your only option is to see if changing any configuration values prompts the device to report on its own:

https://products.z-wavealliance.org/products/1887/configs

Reply:
Michal and Mariusz were changing configuration, but it device did not response with Watts. As I know this is the way they figured out to get Watts update.

Reply:
I checked this with parameters. In configure() I added:
`else if (zwaveInfo.mfr == "014F" && zwaveInfo.prod == "5053" && zwaveInfo.model == "3531") { //GoControl
    	result << response(encap(zwave.configurationV1.configurationSet(parameterNumber: 11, size: 1, scaledConfigurationValue: 1)))
        result << response(encap(zwave.configurationV1.configurationSet(parameterNumber: 12, size: 2, scaledConfigurationValue: 255)))
    }`

Watts didn't update. Also when I set `runLocally` to `true` ours solution didn't work. 
@greens maybe you have another idea for this?

Reply:
@mckeed What can we do about this beyond just setting up polling?

Reply:
@MGoralczykS have you tested while changing the power draw on the switch? In case does report power (watts) but only if it has changed significantly?

If it never reports power on its own then yet we'll have to poll like is done for GE devices.

Reply:
@mckeed yes. I tested this with connected devices and this plug didn't update watts. It only updates when I used refresh button (ST Classic)